internal/pkg/plugin/gitlabcedocker/gitlabcedocker.go

```diff
@@ -1,22 +1,26 @@
 package gitlabcedocker
 
 import (
-	"fmt"
 	"strings"
 )
 
 const (
 	gitlabImageName = "gitlab/gitlab-ce"
 	// TODO expose image tag to user in config file to customize
```

Comment:
Please remove "todo" here.
Then commit again and "rebase" && "force push" to ensure the commit message history is clean.
If it is difficult for you to do that, please ask me for help. ❤️

Reply:
@aFlyBird0   I've completed